app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt

```diff
@@ -177,7 +177,7 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputName_inputPin_clickCreate_checkOpensProfileActivity() {
+  fun testProfileChooserActivity_inputName_inputPin_clickCreate_checkOpensProfileActivity() {
```

Comment:
Revert this change.

Reply:
Rename to `testAddProfileActivity_inputName_inputPin_clickCreate_checkOpensProfileChooserActivity`